kickstart/provision_atomic.erb

```diff
@@ -0,0 +1,40 @@
+<%#
+kind: provision
+name: Atomic Kickstart
+oses:
+- CentOS-Atomic 7
+- Fedora-Atomic 22
+- RedHat-Atomic 7
+%>
+lang <%= @host.params['lang'] || 'en_US.UTF-8' %>
+keyboard <%= @host.params['keyboard'] || 'us' %>
+timezone --utc <%= @host.params['time-zone'] || 'UTC' %>
+
+# Partition table should create /boot and a volume atomicos
+<% if @dynamic -%>
+%include /tmp/diskpart.cfg
+<% else -%>
+<%= @host.diskLayout %>
+<% end -%>
+
+
+bootloader --timeout=3
+<% if @host.operatingsystem.name =~ /.*Fedora.*/ -%>
+ostreesetup --nogpg --osname=fedora-atomic --remote=fedora-atomic --url=<%= @host.medium.path %>/content/repo/ --ref=fedora-atomic/f<%= @host.os.major %>/<%= @host.architecture %>/docker-host
+<% else -%>
+ostreesetup --nogpg --osname=rhel-atomic-host --remote=rhel-atomic-host --url=file:///install/ostree --ref=rhel-atomic-host/<%= @host.os.major %>/<%= @host.architecture %>/standard
```

Comment:
RHEL Atomic lives on the Red Hat CDN and for the moment the way to deploy it with Foreman seems to be to download it and mount the image, so `file:///` works. Fedora can use local storage or a remote URL to fetch the tree as it's publicly accessible.